src/interfaces/views.ts

```diff
@@ -87,6 +88,8 @@ export class SiteView {
   local_site: LocalSite;
   @Type(() => LocalSiteRateLimit)
   local_site_rate_limit: LocalSiteRateLimit;
+  @Type(() => Tagline)
+  taglines: Tagline[];
```

Comment:
As per another comment I made on your back-end PR, this should be an `Option<Tagline[]>`. There may be no taglines.